packages/aws-cdk-lib/aws-eks/README.md

```diff
@@ -227,6 +227,28 @@ cluster.addNodegroupCapacity('custom-node-group', {
 });
 ```
 
+To use a custom AMI for the node group, you can set `amiType` to `CUSTOM`.
+A launch template must be provided.
```

Comment:
Related to the comment above, if custom AMI is supported now leaving the `amiType` property unset, we should call that out in the README too.